sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDAFSuite.scala

```diff
@@ -149,6 +149,16 @@ class HiveUDAFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
       }
     }
   }
+
+  test("HiveUDAF with 0 rows throws NPE") {
+    sql("create table abc(a int)")
+    checkAnswer(sql("select histogram_numeric(a,2) from abc"), Row.fromSeq(Seq.fill(1)(null)))
```

Comment:
nit: `Row(null)` instead of `Row.fromSeq(Seq.fill(1)(null))`

Reply:
Neat. Done